### PR TITLE
Project updates based on copier-ProjectScaffolding

### PR DESCRIPTION
**Pull request by davidbrownell**

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -1,84 +1,87 @@
...
-
-    # - run: |
-    #     echo "Run, Build Application using script"
-    #     ./location_of_script_within_repo/buildscript.sh
-
-    - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v3
-      with:
-        category: "/language:${{matrix.language}}"
+# For most projects, this workflow file will not need changing; you simply need
```
Comment (davidbrownell) [inline .github/workflows/codeql.yml:1]: Note that most of the changes in this file (and other files) have to do with line endings. The original files had "\\r\\n" while the updated files have "\\n".

File: .github/workflows/standard.yml [hunk 2]
```diff
@@ -1,185 +1,202 @@
...
-
-
-    name: Publish
-
-    uses: davidbrownell/dbrownell_DevTools/.github/workflows/callable_publish_python.yaml@CI-v0.17.0
-    with:
-      release_sources_configuration_filename: .github/release_sources.yaml
-    secrets:
-      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+name: "Standard"
```
Comment (varun646) [inline .github/workflows/standard.yml:1]: Just making sure removing the copyright from the top of the file was intentional

File: Build.py [hunk 3]
```diff
@@ -1,86 +1,79 @@
...
-    app,
-)
-
-
-# ----------------------------------------------------------------------
-# ----------------------------------------------------------------------
-# ----------------------------------------------------------------------
-if __name__ == "__main__":
-    sys.exit(app())
+"""Build tasks for this python project."""
```
Comment (varun646) [inline Build.py:1]: same here with the copyright comment

File: pyproject.toml [hunk 4]
```diff
@@ -1,187 +1,186 @@
...
-]
-
-# ----------------------------------------------------------------------
-# |
-# |  setuptools
-# |
-# ----------------------------------------------------------------------
-[tool.setuptools.dynamic]
-version = { attr = "RepoAuditor.__version__" }
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
```
Comment (varun646) [inline pyproject.toml:1]: same here with the copyright comment

File: src/BuildBinary.py [hunk 5]
```diff
@@ -1,104 +1,93 @@
...
-                "tkinter",
-            ],
-            "no_compress": False,
-            "optimize": 0,
-            # "packages": [],
-            # "include_files": [],
-        },
-    },
-)
+"""Builds the binary for this project."""
```
Comment (varun646) [inline src/BuildBinary.py:1]: same here with the copyright comment

File: src/RepoAuditor/__init__.py [hunk 6]
```diff
@@ -1,14 +1,8 @@
...
-# ----------------------------------------------------------------------
-# pylint: disable=missing-module-docstring,invalid-name
-
-APP_NAME = "RepoAuditor"
-
-# Note that this value will be overwritten by calls to `python ../../Build.py update_version` based
-# on changes observed in the git repository. The default value below will be used until the value
-# here is explicitly updated as part of a commit.
-__version__ = "0.1.0"
+# pylint: disable=missing-module-docstring,invalid-name
```
Comment (varun646) [inline src/RepoAuditor/__init__.py:1]: same here with the copyright comment

**Review by varun646 — APPROVED** [review]

**Review by krachwal — APPROVED** [review]

**Pull request merged**